yarn.lock

```diff
@@ -9025,6 +9059,37 @@ vm-browserify@^1.0.1:
   resolved "https://registry.yarnpkg.com/vm-browserify/-/vm-browserify-1.1.2.tgz#78641c488b8e6ca91a75f511e7a3b32a86e5dda0"
   integrity sha512-2ham8XPWTONajOR0ohOKOHXkm3+gaBmGut3SRuu75xLd/RRaY6vqgh8NBYYk7+RW3u5AtzPQZG8F10LHkl0lAQ==
 
+vscode-json-languageservice@^3.5.1:
```

Comment:
this is specific to your IDE... might not be needed here.

Reply:
I looked in to it, it seems like it has to do with the eslint json plugin, it's a dependency they include, which i guess isn't abt my ide specifically. I left it in anyway, since it's a dependency listed by one of our project dependencies.